paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/AppendOnlyTableITCase.java

```diff
@@ -85,6 +85,22 @@ public void testReadWrite() {
         assertThat(rows).containsExactlyInAnyOrder(Row.of("AAA"), Row.of("BBB"));
     }
 
+    @Test
+    public void testReadWriteWithUnwareBucket() throws Exception {
```

Comment:
What is this test for?

Reply:
Sorry, I have change the method name to testReadUnwareBucketTableWithRebalanceShuffle.

This test is to test that the data read under the following conditions is correct：
1、consumer.mode=exactly-once
2、Streaming read unware bucket table.
3、Send Split from MonitorFunction to ReadOperator is rebalance.